nemo/collections/llm/recipes/llama3_8b.py

```diff
@@ -96,7 +95,10 @@ def pretrain_recipe(
 
 
 def hf_resume() -> Config[nl.AutoResume]:
-    return Config(nl.AutoResume, import_path="hf://meta-llama/Meta-Llama-3.1-8B")
+    return Config(
+        nl.AutoResume,
+        selective_restore_config=Config(nl.SelectiveRestoreConfig, path="hf://meta-llama/Meta-Llama-3.1-8B"),
```

Comment:
This didn't originate from this PR, but this hf path should be `hf://meta-llama/Meta-Llama-3-8B` to match the `Llama3Config8B` above. (Alternatively change above to `Llama31Config8B`, and same for other recipe files.

Reply:
let me know if you think this is okay, i can also make the change

Reply:
Oh yeah feel free to change, I must've missed this.

Reply:
Updated.